terasoluna-gfw-web/src/test/java/org/terasoluna/gfw/web/token/transaction/TransactionTokenIssue364InterceptorTest.java

```diff
@@ -0,0 +1,485 @@
+/*
```

Comment:
Why did you separate test class?
In addition the name of test class should not include issue number. The name should be generic for its maintainability.